airflow/docker.go

```diff
@@ -1461,23 +1406,6 @@ var CheckTriggererEnabled = func(imageLabels map[string]string) (bool, error) {
 	return versions.GreaterThanOrEqualTo(runtimeVersion, triggererAllowedRuntimeVersion), nil
 }
 
-// CheckM1Image checks if the CLI is currently running on M1 architecture
-// next it checks if the runtime version of the image building is above 6.0.4
-// if the image is M1 architecture and runtime version is less than or equal to 6.0.4 it will print true
-var CheckM1Image = func(imageLabels map[string]string) bool {
-	if !isM1(runtime.GOOS, runtime.GOARCH) {
-		// the architecture is not arm64 no need to print message
-		return false
-	}
-	runtimeVersion, ok := imageLabels[runtimeVersionLabelName]
-	if !ok {
-		// cannot determine runtime version print message by default
-		return true
-	}
-
-	return versions.LessThanOrEqualTo(runtimeVersion, M1ImageRuntimeVersion)
-}
```

Comment:
This function is purely used to decide if the `astro dev start --wait` flag defaults to `1m` or `5m`. Thinking we just simplify and use `5m`. The CLI progresses as soon as it sees the `200` from the webserver `/health` endpoint.